### PR TITLE
Pin and document what version of yarn to use

### PR DESCRIPTION
**Pull request by fedde-s**
Using the ‘project-locked’ that's recommended in [the yarn 2.x docs](https://yarnpkg.com/getting-started/install#global-install), but
also documented for use with yarn classic, could make it easier to stay
on a consistent version across the team without requiring any changes to
the global yarn installs that we were using already.

### CODE REVIEW

**Comment by fedde-s** [comment]
@andrewhercules or others, if you feel that this is overkill for now, do feel free to let me know that too!

**Comment by fedde-s** [comment]
Hm, looks like CI is failing at the first `yarn` command, but that same error occurs in the base `alpha` branch.

**Comment by fedde-s** [comment]
Pull request #133 should address the build failure, so that we can be assured that this change doesn't break anything.

File: README.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ To customise the platform app:
 - Create a directory somewhere outside this repository where you will put your customisation files.
 - Add an environment variable called `$CUSTOMISATIONS_DIR` that points to the directory created in the step above.
 - Inside your customisation directory you can overwrite or add new files following the same directory structure as `src/public`.
+- Make sure you have [yarn](https://classic.yarnpkg.com/en/docs/install) installed.
+  Any version below 2.0, but no older than 1.0,
+  will defer to the appropriate version while working in this repository.
+  Run `yarn install` in the repository to download dependencies.
```
Comment (fedde-s) [inline README.md:67]: Maybe we should include the `--frozen-lockfile` option here, to ensure that anyone who isn't actively updating the dependencies gets the versions that the developers tested.

Comment (fedde-s) [reply README.md:67]: Never mind, after taking another look at [the docs](https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install-frozen-lockfile) I think that only adds another way for the build process to fail, which is good for CI but wouldn't give the best impression to people who've just checked out the codebase for the first time.

**Comment by javfg** [comment]
Given that we are using the latest yarn version, does it make sense to include the release in the code? It will probably be updated as development continues.

**Comment by fedde-s** [comment]
This approach allows updating it for everyone by running `yarn set version <new version>`, without having to ask all core developers, contributors and customisation builders to change their global installation.

**Comment by fedde-s** [comment]
And just like the pins in `yarn.lock` for all other dependencies, it should avoid some people running into bugs that others can't reproduce because they're using different versions.

in an extreme example, I already lost some time looking up how to install the latest version of yarn, seeing it fail to build, and installing yarn classic instead.

**Comment by javfg** [comment]
Agreed. Thanks for the PR!

**Pull request merged**